SDWebImage/SDImageCache.m

```diff
@@ -270,6 +254,27 @@ - (void)storeImage:(UIImage *)image forKey:(NSString *)key toDisk:(BOOL)toDisk {
     [self storeImage:image recalculateFromImage:YES imageData:nil forKey:key toDisk:toDisk];
 }
 
+- (void)storeImageDataToDisk:(NSData *)imageData forKey:(NSString *)key {
+    
+    if (!imageData) return;
```

Comment:
Ok no problem, I'll align to your coding style. Is there a doc somewhere where the style to adhere to is defined?